R/applications.R

```diff
@@ -133,7 +133,7 @@ showLogs <- function(appPath = getwd(), appFile = NULL, appName = NULL,
                      account = NULL, entries = 50, streaming = FALSE) {
 
   # determine the log target and target account info
-  target <- deploymentTarget(appPath, appName, account)
+  target <- deploymentTarget(appPath, appName, NULL, account)
```

Comment:
Fixed now (via adopting a canonical ordering for the arguments as @aronatkins suggested). Nice catch, thanks!